llvm/test/Transforms/Util/loop-iter-count-offset-assumptions.ll

```diff
@@ -0,0 +1,209 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+;
+; This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+; See https://llvm.org/LICENSE.txt for license information.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+;
+; (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+; unit test for the loop iteration metadata conversion to assumptions
+;
+; NOTE: Maintain assumptions through correlated value propagation
+;
+; REQUIRES: aie2
```

Comment:
This gives to me `UNSUPPORTED: LLVM :: Transforms/Util/loop-iter-count-offset-assumptions.ll (1 of 1)` even having this target registered.

Reply:
I removed it from the PR, since Correlated Value Propagation should be rather fixed, then introducing an (accepted) hotfix